app/templates/flatpages/about.html

```diff
@@ -0,0 +1,111 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>About | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<space class="large"></space>
+<div class="content" style="margin-bottom: 90px;">
+  <h1 style="text-align: center">This is HackItFoward</h1>
+  <div style="border-radius: 20px; background-color: white; height:50%; margin-top: 50px; text-align: center;">
```

Comment:
You can remove at least 3 of these with the `bg-white`, `h-50`, `u-text-center`. Please check the rest of your styles as well (such as the image below).